deploy/helm/metrics-collector-config.yaml

```diff
@@ -11,6 +11,55 @@ extensions:
 {{ toYaml .Values.otel.metrics.memory_ballast | indent 4 }}
 
 processors:
+{{- if .Values.otel.metrics.k8s_instrumentation.enabled }}
+  k8sattributes:
+    auth_type: "serviceAccount"
+    passthrough: false
+    extract:
+      metadata:
+        - k8s.deployment.name
+        - k8s.replicaset.name
+        - k8s.daemonset.name
+        - k8s.job.name
+        - k8s.cronjob.name
+        - k8s.statefulset.name
+{{- if .Values.otel.metrics.k8s_instrumentation.annotations.enabled }}
+      annotations:
+        - key_regex: (.*)
+          tag_name: k8s.pod.annotations.$$1
+          from: pod
+        - key_regex: (.*)
+          tag_name: k8s.namespace.annotations.$$1
+          from: namespace
+{{- end }}
+{{- if .Values.otel.metrics.k8s_instrumentation.labels.enabled }}
+      labels:
+        - key_regex: (.*)
```

Comment:
Do we really want to add annotations/labels to all metrics? Isn't that too much? What's benefit to have it on all instead of create some helper metric which has just annotation purpose?

Reply:
We need this on custom dashboards (and maybe also Alerts and metric explorer), being able to create custom widget, selecting for example `k8s.container_cpu_usage_seconds_total` and filter datapoints based on labels. So we must assign labels to all those metrics. I hope it won't be too much, in sample data it looks like only 10% of increase, but we will see when we test this in dev-ssp